Source/Util/Command.js

```diff
@@ -16,9 +16,9 @@ class Command {
     }
 
     userCanAccess(member) {
-        let isAdmin = member.client.config.admins.includes(member.id);
+        var isAdmin = member.client.config.admins.includes(member.id);
         if(this.requiresAdmin && !isAdmin) return false;
-        if(member) return (member.hasPermissions(this.requiredPermissions)) || isAdmin;
+        if(member) return (member.hasPermission(this.requiredPermissions)) || isAdmin;
```

Comment:
It was changed in the latest version of discord.JS, hasPermissions is deprecated because hasPermission accepts an array